CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 
 ### New Features
 - Init Gradle project. (#1 #2)
+- Migrate contract package of `iexec-common` library and the other classes used in the POCO protocol to this `iexec-commons-poco` library. (#3) 
```

Comment:
Very minor
```suggestion
- Migrate from `iexec-common` library to this `iexec-commons-poco` library packages related to blockchain and iExec PoCo protocol. (#3) 
```
You can keep yours or another one no problem :+1:

Reply:
ok,it's integrated